src/Site.js

```diff
@@ -830,7 +854,24 @@ Site.prototype.copyFontAwesomeAsset = function () {
  * Copies MarkBind assets to the assets folder
  */
 Site.prototype.copyMarkBindAsset = function () {
-  return fs.copyAsync(this.siteAssetsSrcPath, this.siteAssetsDestPath);
+  const maybeOverrideDefaultBootstrapTheme = () => {
+    const { theme } = this.siteConfig;
+    if (!theme || !Object.prototype.hasOwnProperty.call(SUPPORTED_THEMES_PATHS, theme)) {
```

Comment:
Could this check be simplified to [`_.has(SUPPORTED_THEMES_PATHS, theme)`](https://lodash.com/docs/4.17.11#has)?

Reply:
Good call, that definitely looks more readable. Done 👍